src/plugins/plugin.tooltip.js

```diff
@@ -544,6 +562,9 @@ export class Tooltip extends Element {
       };
     }
 
+    me._tooltipItems = tooltipItems;
+    me.$context = undefined;
```

Comment:
Is the context actually cached at all, or is it always re-created?

Reply:
Yes, it's cached in the `getContext()` call.